persistence/sql/identity/persister_identity.go

```diff
@@ -1055,6 +1059,78 @@ func (p *IdentityPersister) UpdateIdentityColumns(ctx context.Context, i *identi
 	return nil
 }
 
+func (p *IdentityPersister) ListIdentitiesSCIMData(ctx context.Context, params identity.ListIdentitySCIMDataParameters) (identities []identity.Identity, total int, err error) {
+
+	ctx, span := p.r.Tracer(ctx).Tracer().Start(ctx, "persistence.sql.ListIdentitiesSCIMData",
+		trace.WithAttributes(attribute.Stringer("network.id", p.NetworkID(ctx))))
+	defer otelx.End(span, &err)
+
+	var (
+		query               strings.Builder
+		args                []any
+		nid                 = p.NetworkID(ctx)
+		identitiesWithTotal = make([]struct {
+			identity.Identity
+			Total int `db:"total"`
+		}, 0)
+	)
+
+	if err = p.Transaction(ctx, func(ctx context.Context, con *pop.Connection) error {
+		query.WriteString(`
+SELECT identities.*, COUNT(*) OVER() AS total
+FROM identities
+WHERE identities.nid = ?
```

Comment:
If all filters are empty/unset, this will read all rows in the identities table matching the filter here